blockchain/v2/io.go

```diff
@@ -59,7 +64,11 @@ func (sio *switchIO) sendStatusResponse(height int64, peerID p2p.ID) error {
 	if peer == nil {
 		return fmt.Errorf("peer not found")
 	}
-	msgBytes := cdc.MustMarshalBinaryBare(&bcStatusResponseMessage{Height: height})
+
+	msgBytes, err := bc.EncodeMsg(&bcproto.StatusResponse{Height: height})
```

Comment:
@erikgrinaker is this correct? it's not filling Base

Reply:
No, that doesn't seem right. Not sure what happened here, I'll add a separate PR.